apps/samourai-server/nginx/connect/index.html

```diff
@@ -82,6 +82,7 @@ <h2 class="text">How to connect Whirpool</h2>
             <li class="step">Click “Mix”.</li>
             <li class="step">Congratulations! Whirpool is now mixing your UTXOs on your Umbrel!</li>
         </ol>
+        <p><strong>Note:</strong> You'll need to open Whirlpool GUI and re-enter your password to continue mixing after restarting your Umbrel or doing an OTA update.</p>
```

Comment:
Can we change `after restarting your Umbrel or doing an OTA update` to `after restarting or updating your Umbrel` to make sure the word "OTA" doesn't confuse users and there's no ambiguity around the update ("Umbrel update or Whirpool update?").

Reply:
Good point